packages/now-cli/src/util/output/create-output.ts

```diff
@@ -22,14 +22,15 @@ export default function createOutput(opts?: OutputOptions) {
 }
 
 function _createOutput({ debug: debugEnabled = false }: OutputOptions = {}) {
+  let spinnerMessage = '';
   let spinner: StopSpinner | null = null;
 
   function isDebugEnabled() {
     return debugEnabled;
   }
 
   function print(str: string) {
-    stopSpinner();
+    if (spinnerMessage) stopSpinner();
```

Comment:
Why conditionally call `stopSpinner()`? Seems like you could handle that in the `stopSpinner()` function.

Reply:
This one probably isn't necessary anymore.